lib/onceler/recorder.rb

```diff
@@ -63,9 +68,21 @@ def record!
       @recordings.each do |recording|
         recording.prepare_medium!(@tape)
       end
-      @recordings.each do |recording|
-        recording.record_onto!(@tape)
+
+      # wrap the before in a lambda
+      stack = -> do
+        @recordings.each do |recording|
+          recording.record_onto!(@tape)
+        end
+      end
+      # and then stack each around block on top
+      @arounds.each do |around|
```

Comment:
should this be `@arounds.reverse` so it calls them in order still? not sure what rspec does but we should probably match that.